kafka-streams/src/main/java/io/micronaut/configuration/kafka/streams/KafkaStreamsFactory.java

```diff
@@ -63,6 +64,9 @@ public class KafkaStreamsFactory implements Closeable {
 
     private final ApplicationEventPublisher eventPublisher;
 
+    @Value("${kafka.streams-close-seconds:3}")
```

Comment:
this should be in in a `@ConfigurationProperties` object with the default value there.